### PR TITLE
point docs to core docs site

### PR DESCRIPTION
**Pull request by rlh1994**
## Description & motivation
Docs moved to https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: docs/markdown/snowplow_mobile_overview.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 # Snowplow Mobile Package
 
-Welcome to the documentation site for the Snowplow mobile dbt package. The package contains a fully incremental model, that transforms raw mobile event data generated by the [Snowplow iOS][ios-tracker] or [android trackers][android-tracker] into a series of derived tables of varying levels of aggregation.
+Welcome to the model documentation site for the Snowplow mobile dbt package. The package contains a fully incremental model, that transforms raw mobile event data generated by the [Snowplow iOS][ios-tracker] or [android trackers][android-tracker] into a series of derived tables of varying levels of aggregation.
 
-**Note this doc site is linked to latest release of the package. If you are not using the latest release, [generate and serve](https://docs.getdbt.com/reference/commands/cmd-docs#dbt-docs-serve) the doc site locally for accurate documentation.**
+**For more information, including a QuickStart guide, operation and configuration, and implementing your own custom modules on top of this please visit the [Snowplow Docs](https://docs.snowplowanalytics.com/docs/modeling-your-data/modeling-your-data-with-dbt/).**
```
Comment (paulboocock) [inline docs/markdown/snowplow_mobile_overview.md:9]:
```suggestion
**For more information, including a QuickStart guide, operation and configuration, and implementing your own custom modules on top of this please visit the [Snowplow Docs](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/).**
```

**Review by paulboocock — CHANGES_REQUESTED** [review]

File: docs/markdown/snowplow_mobile_overview.md (hunk 1 above)
Comment (rlh1994) [reply docs/markdown/snowplow_mobile_overview.md]: Replaced all instances to snowplow.io

**Comment by paulboocock** [comment]
Looks good, but would be good to figure out whats going on with the tests. Perhaps the same issue as with the `web` package PR?

**Comment by emielver** [comment]
> Looks good, but would be good to figure out whats going on with the tests. Perhaps the same issue as with the `web` package PR?

@paulboocock it seems to be a bug in dbt v1.3.0, trying to get together enough info for a bug report on their repo but will likely take a bit to sort out since they're in full Coalesce mode atm. We're hardcoding dbt v1.2.x in the tests now and that's passing fine

**Pull request merged**